api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -99,4 +99,24 @@ message QuicProtocolOptions {
   // QUIC layer by replying with an empty version negotiation packet to the
   // client.
   bool reject_new_connections = 13;
+
+  // Enable ``JA3`` fingerprinting for QUIC connections.
+  //
+  // When enabled, the ``JA3`` hash will be computed from the ``ClientHello`` message
+  // and stored in the connection info for access by access logs and other components.
+  // ``JA``3 is a standard method of creating SSL client fingerprints that are easy to
+  // produce and can be used for identifying clients across multiple TLS sessions.
+  //
+  // Defaults to ``false``.
+  google.protobuf.BoolValue enable_ja3_fingerprinting = 14;
```

Comment:
Do you think this field would ever need to be default true?  If not, what's the advantage of using the wrapped scalar instead of just `bool`?